### PR TITLE
Standardize naming of preds/probs

### PR DESCRIPTION
**Pull request by bhancock8**
- Standardize names of label types: golds, preds, and probs (instead of gold, pred, prob).
- Remove an extra kwarg in metric_score (had both prob and probs, which collided after the refactor)

Test plan:
All tests still pass after refactor

### CODE REVIEW

**Comment by henryre** [comment]
> All tests still pass after refactor

but do they tho?

**Comment by bhancock8** [comment]
They do indeed. Ready for review!

File: snorkel/slicing/utils.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def convert_to_slice_tasks(base_task: Task, slice_names: List[str]) -> List[Task
         )
         pred_task_flow = body_flow + [pred_transform_op, pred_head_op]
 
-        # Create pred task
+        # Create preds task
```
Comment (henryre) [inline snorkel/slicing/utils.py]: !!!

File: snorkel/labeling/model/label_model.py [hunk 2]
```diff
@@ -242,7 +242,7 @@ def get_conditional_probs(self, source=None):
             c_probs[i * (self.k + 1) + 1 : (i + 1) * (self.k + 1), :] = mu_i
 
             # The 0th row (corresponding to abstains) is the difference between
-            # the sums of the other rows and one, by law of total prob
+            # the sums of the other rows and one, by law of total probs
```
Comment (henryre) [inline snorkel/labeling/model/label_model.py]: !!!

File: snorkel/labeling/analysis.py [hunk 3]
```diff
@@ -145,7 +145,7 @@ def lf_summary(
     Args:
         L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
             jth LF to the ith candidate
-        Y: an [n] or [n, 1] np.ndarray of gold labels.
+        Y: an [n] or [n, 1] np.ndarray of golds labels.
```
Comment (henryre) [inline snorkel/labeling/analysis.py]: !!!

File: snorkel/labeling/analysis.py [hunk 4]
```diff
@@ -125,7 +125,7 @@ def lf_empirical_accuracies(L: Matrix, Y: ArrayLike) -> np.ndarray:
     Args:
         L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
             jth LF to the ith candidate
-        Y: an [n] or [n, 1] np.ndarray of gold labels
+        Y: an [n] or [n, 1] np.ndarray of golds labels
```
Comment (henryre) [inline snorkel/labeling/analysis.py]: !!!

File: snorkel/analysis/metrics.py [hunk 5]
```diff
@@ -115,27 +123,35 @@ def recall_score(
...
     Calculate recall for a single class.
     Args:
-        gold: A 1d array-like of gold labels
-        pred: A 1d array-like of predicted labels (assuming abstain = 0)
-        prob: Unused (kept for compatibility with expected metric func signature)
-        ignore_in_gold: A list of labels for which elements having that gold
+        golds: A 1d array-like of golds labels
+        preds: A 1d array-like of predicted labels (assuming abstain = 0)
+        probs: Unused (kept for compatibility with expected metric func signature)
+        ignore_in_gold: A list of labels for which elements having that golds
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

File: snorkel/analysis/metrics.py [hunk 6]
```diff
@@ -115,27 +123,35 @@ def recall_score(
...
             label will be ignored.
-        ignore_in_pred: A list of labels for which elements having that pred
+        ignore_in_pred: A list of labels for which elements having that preds
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

File: snorkel/analysis/metrics.py [hunk 7]
```diff
@@ -115,27 +123,35 @@ def recall_score(
 
 
 def fbeta_score(
-    gold, pred, prob=None, pos_label=1, beta=1.0, ignore_in_gold=[], ignore_in_pred=[]
+    golds,
+    preds,
+    probs=None,
+    pos_label=1,
+    beta=1.0,
+    ignore_in_gold=[],
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

File: snorkel/analysis/metrics.py [hunk 8]
```diff
@@ -115,27 +123,35 @@ def recall_score(
...
+    ignore_in_pred=[],
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

File: snorkel/analysis/metrics.py [hunk 9]
```diff
@@ -82,27 +88,29 @@ def precision_score(
...
+    golds, preds, probs=None, pos_label=1, ignore_in_gold=[], ignore_in_pred=[]
 ):
     """
     Calculate recall for a single class.
     Args:
-        gold: A 1d array-like of gold labels
-        pred: A 1d array-like of predicted labels (assuming abstain = 0)
-        prob: Unused (kept for compatibility with expected metric func signature)
-        ignore_in_gold: A list of labels for which elements having that gold
+        golds: A 1d array-like of golds labels
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

File: snorkel/analysis/metrics.py [hunk 10]
```diff
@@ -3,73 +3,79 @@
...
+    golds, preds, probs=None, pos_label=1, ignore_in_gold=[], ignore_in_pred=[]
 ):
     """
     Calculate precision for a single class.
     Args:
-        gold: A 1d array-like of gold labels
-        pred: A 1d array-like of predicted labels (assuming abstain = 0)
-        prob: Unused (kept for compatibility with expected metric func signature)
-        ignore_in_gold: A list of labels for which elements having that gold
+        golds: A 1d array-like of golds labels
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

File: snorkel/analysis/metrics.py [hunk 11]
```diff
@@ -3,73 +3,79 @@
...
-def coverage_score(gold, pred, prob=None, ignore_in_gold=[], ignore_in_pred=[]):
+def coverage_score(golds, preds, probs=None, ignore_in_gold=[], ignore_in_pred=[]):
     """
     Calculate (global) coverage.
     Args:
-        gold: A 1d array-like of gold labels
-        pred: A 1d array-like of predicted labels (assuming abstain = 0)
-        prob: Unused (kept for compatibility with expected metric func signature)
-        ignore_in_gold: A list of labels for which elements having that gold
+        golds: A 1d array-like of golds labels
```
Comment (henryre) [inline snorkel/analysis/metrics.py]: !!!

**Comment by henryre** [comment]
@bhancock8 can we close?

**Pull request closed without merging**